server/routes/bookedtoday/arrivals/confirmArrival/index.ts

```diff
@@ -58,12 +62,14 @@ export default function routes(services: Services): Router {
       '/prisoners/:id/check-answers',
       checkNewArrivalPresent,
       redirectIfDisabledMiddleware(config.confirmEnabled),
+      addLockId,
       checkAnswersController.view()
```

Comment:
the `addLockId` on the `get` generates a new lock identifier/key on every render/reload.

the lock is embedded as a hidden input and submitted to the POST controller. 
Double clicks will send the same lock identifier multiple times.    

The `obtainLock` middleware will fail for subsequent requests passing the same id and redirect to the arrival list page.